assets/js/components/Vehicle.vue

```diff
@@ -37,11 +63,37 @@ export default {
 		vehicleSubline: function () {
 			return this.collectProps(VehicleSubline);
 		},
+		// TODO: Handle language support
+		vehicleLoginButtonText: function () {
+			if (this.vehicleProviderLoginPath !== "") {
+				return "Login";
+			}
+
+			return "";
+		},
 	},
 	methods: {
 		targetSocUpdated: function (targetSoC) {
 			this.$emit("target-soc-updated", targetSoC);
 		},
+		providerLogin: async function () {
+			await axios
```

Comment:
Siehe axios Errorhandling in master, kann vmtl vereinfacht werden wenn Du den Defaulthandler nutzt